src/aave-v3/SupplyVault.sol

```diff
@@ -42,14 +42,14 @@ contract SupplyVault is ISupplyVault, SupplyVaultBase {
     /// STRUCTS ///
 
     struct UserRewardsData {
-        uint128 index; // User rewards index for a given reward token (in wad).
+        uint128 index; // User rewards index for a given reward token (in ray).
         uint128 unclaimed; // Unclaimed amount for a given reward token (in reward tokens).
     }
 
     /// CONSTANTS AND IMMUTABLES ///
 
-    uint256 public constant SCALE = 1e36;
     IRewardsManager public immutable rewardsManager; // Morpho's rewards manager.
+    uint256 constant RAY = 1e27;
```

Comment:
```suggestion
    uint256 public constant RAY = 1e27;
```

Reply:
And add `RAY()` automatic getter to the interface in this case

Reply:
it's public by default @Rubilmax but I'm ok with the change

Reply:
This change is not mandatory however the change in the interface is IMO
Also making it explicit is required by some linters (but I can't remember which one I seem to have disabled the one which was telling me to make variable visibility explicitly)

Reply:
We must do it yes